dist/index.js

```diff
@@ -136,35 +131,35 @@ async function run() {
       const apiMsgs = await getCommitMessages("notification-api", oldApiSha)
 
       let logs = `ADMIN: \n\n ${adminMsgs.join("\n")} \n\n API: \n\n ${apiMsgs.join("\n")}`
-      if (await hasChangesOnStagingTfRepo()) {
-        logs = `⚠️ There was a merged pull request in [notification-staging-tf](https://github.com/cds-snc/notification-staging-tf) since last deploy in production! Check first that you don't need to change the infrastructure here. \n\n ${logs}`
+      if (await isNotLatestVersion()) {
+        logs = `⚠️ The production version of manifests is behind the latest staging version. Consider upgrading to the latest version before! \n\n ${logs}`
```

Comment:
Suggestion:
Consider upgrading to the latest version before {merging this pull request}.

Reply:
Cheers, just changed this